### PR TITLE
add faq info about bundle size

### PR DESCRIPTION
**Pull request by camilamacedo86**
add faq info about bundle size

### CODE REVIEW

File: website/content/en/docs/faqs/_index.md [hunk 1]
```diff
@@ -230,3 +230,13 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
...
+
+## What is the bundle limit size? Was this amount increased?
+
+Bundles have a size limitation because their manifests are used to create a configMap, and the Kubernetes API does not 
+allow configMaps larger than `~1MB`. However, from [OLM](https://github.com/operator-framework/operator-lifecycle-manager) version `v0.19.0` and [OPM](https://github.com/operator-framework/operator-registry) `1.17.5`, these values were increased to `~4MB` because we are compressing them. ([More info](https://github.com/operator-framework/operator-registry/pull/685)).
+
+The change to allow bigger bundles from [OLM](https://github.com/operator-framework/operator-lifecycle-manager) version `v0.19.0` only impacts the full bundle size amount. 
+Any single manifest within the bundle such as the CRD will still make the bundle uninstallable if it exceeds the default file size limit on clusters (`~1MB`).
+
+_NOTE: Openshift k8S vendor began to allow `~4MB` uncompressed bundle sizes from its `4.9` version release version._
```
Comment (asmacdo) [inline website/content/en/docs/faqs/_index.md:242]: We should not mention OpenShift

**Review by asmacdo — CHANGES_REQUESTED** [review]
lgtm other than the removal of the note about OpenShift versions.

File: website/content/en/docs/faqs/_index.md (hunk 1 above)
Comment (camilamacedo86) [inline website/content/en/docs/faqs/_index.md:242]:
```suggestion
```

**Review by asmacdo — APPROVED** [review]
/lgtm

**Pull request merged**